electronic_store/demo/website_view.xml

```diff
@@ -0,0 +1,1355 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <template id="homepage" name="Home">
+        <t name="Homepage" t-name="website.homepage">
+            <t t-call="website.layout">
+                <t t-set="pageName" t-value="'homepage'"/>
+                <div id="wrap" class="oe_structure oe_empty">
+                    <section class="s_cover parallax s_parallax_is_fixed bg-black-50 pt96 pb96 o_full_screen_height o_cc o_cc5 o_colored_level" data-scroll-background-ratio="1" data-snippet="s_cover" style="background-image: none;" data-name="Cover">
+                        <span class="s_parallax_bg oe_img_bg o_bg_img_center" style="background-image: url('/web/image/electronic_store.ir_attachment_471'); background-position: 50% 0px;"/>
+                        <div class="o_we_bg_filter bg-black-25"/>
+                        <div class="container s_allow_columns">
+                            <div class="row">
+                                <div class="o_colored_level col-lg-6">
+                                    <h1 style="text-align: left">&amp; Festival Seasons</h1>
+                                    <p class="lead" style="text-align: left">
+                                        After a year's absence, the free and independent festival is back.
+                                    </p>
+                                    <p class="lead" style="text-align: left">
+                                        More than ever young talents are honoured on the big stage with more famous artists.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                </div>
+            </t>
+        </t>
+    </template>
+    
+    <record id="ir_ui_view_2542" model="ir.ui.view">
```

Comment:
I don't see the added value of this view. What did you change?

Reply:
So the consultant has only made changes in the text so I have made changes accordingly